src/Symfony/Component/Yaml/Inline.php

```diff
@@ -774,8 +774,8 @@ private static function parseTag($value, &$i, $flags)
         $nextOffset += strspn($value, ' ', $nextOffset);
 
         // Is followed by a scalar
-        if (!isset($value[$nextOffset]) || !in_array($value[$nextOffset], array('[', '{'), true)) {
-            // Manage scalars in {@link self::evaluateScalar()}
+        if ((!isset($value[$nextOffset]) || !in_array($value[$nextOffset], array('[', '{'), true)) && 'tagged' !== $tag) {
```

Comment:
opting out from a deprecation in the specific required case (`!tagged foo` was a string before this, not yaml compliant so deprecated anyway)